pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -244,6 +244,15 @@ type PersistentVolumeController struct {
 func (ctrl *PersistentVolumeController) syncClaim(claim *v1.PersistentVolumeClaim) error {
 	klog.V(4).Infof("synchronizing PersistentVolumeClaim[%s]: %s", claimToClaimKey(claim), getClaimStatusForLogging(claim))
 
+	// Set correct "migrated-to" annotations on PVC and update in API server if
+	// necessary
+	claim, err := ctrl.updateClaimMigrationAnnotations(claim)
```

Comment:
Might need to check with @saad-ali but I think the space shuttle pattern has us assigning a new var `updatedClaim` and then explicitly writing `claim = updatedClaim` after the if block.

Reply:
I don't really see this pattern elsewhere but it does seem to fit into the philosophy

Reply:
Now I'm wondering where I saw it